docusaurus.config.js

```diff
@@ -25,6 +25,53 @@ module.exports = {
   favicon: 'img/meta/favicon-96x96.png',
   organizationName: 'ionic-team',
   projectName: 'ionic-docs',
+  presets: [
+    [
+      '@docusaurus/preset-classic',
```

Comment:
There seems to be an additional gutter on the right (notice the right margin is larger than the left margin in the `branch` image)

Looks to be a similar issue on an individual component's docs page too


| production | branch |
| - | - |
| ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/3e13ba70-0e11-445f-a92f-6406ef7147e0) | ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/f7342188-1681-4953-b826-98982ea3820f) |

Reply:
Do you remember what width and browser you saw this? I'm not able to replicate it.

Reply:
Sorry! I had my screenshots backwards. It looks like the problem exists in production, but your branch fixes it 🎉